src/e2e-test/java/io/cdap/plugin/bigquery/stepsdesign/BigQueryBase.java

```diff
@@ -224,22 +245,32 @@ public void enterBigQuerySinkPropertyTemporaryBucketName(String temporaryBucket)
     CdfBigQueryPropertiesActions.enterTemporaryBucketName(PluginPropertyUtils.pluginProp(temporaryBucket));
   }
 
+  @Then("Enter BigQuery property reference name {string}")
```

Comment:
Use the existing step to enter the value in any property. Remove this step definition.